build-support/bin/ci.py

```diff
@@ -223,13 +223,10 @@ def pants_command(
                         "--no-v1",
                         "--v2",
                         "--pants-config-files=pants.remote.ini",
-                        # We turn off speculation to reduce the risk of flakiness, where a test
-                        # passes either locally or remotely and fails in the other environment.
-                        "--process-execution-speculation-strategy=none",
                         f"--remote-oauth-bearer-token-path={oauth_token_path}",
                         "test",
                         *sorted(targets),
-                      ],
+                      ] + ["--", "-s"],
```

Comment:
What is this for? Also this can be kept as a part of the list rather than concatenating a new list.

Reply:
Removed it sorry, was for debugging. tells pytest not to suppress output.